build.psm1

```diff
@@ -1862,6 +1862,18 @@ function New-MSIPackage
         [Switch] $Force
     )
 
+    ## need RCEdit to modify the binaries embedded resources
+    if (-not (Test-Path "~/.rcedit/rcedit-x64.exe"))
+    {
+        $rceditUrl = "https://github.com/electron/rcedit/releases/download/v1.0.0/rcedit-x64.exe"
```

Comment:
Can we install this via chocolatey or nuget?  Based on my understanding of our requirements for release builds, this will have to be rewritten sooner rather than later.

Reply:
And move it to Start-PSBootstrap?

Reply:
I don't found nuget package.

Reply:
It's not on chocolatey or nuget currently.  I could move the install to start-psbootstrap.

Reply:
It's under MIT license.  Nothing keeps us from packaging it our self... and chocolatey, doesn't even package it, it just encapsulates the installation code.

Reply:
But this can wait until we have to do it.